src/ecalc/libraries/libecalc/common/tests/core/models/compressor_modelling/test_variable_speed_compressor_train_multiple_streams.py

```diff
@@ -219,6 +219,41 @@ def variable_speed_compressor_train_two_compressors_two_streams(
     )
 
 
+@pytest.fixture
```

Comment:
:+1:

Reply:
great with test. and it seems very minimal. Win! :)

We should have more tests like these, for all the edge cases we can think of. Just a reminder for future and improvement of tests coverage :)